data/core/closetheloop.md

```diff
@@ -726,11 +726,11 @@
     - action_chitchat
 * ask_howold
     - action_chitchat
-* out_of_scope{"number":42}
-    - utter_out_of_scope
```

Comment:
@ArjaanBuijk let's go ahead and add `use_entities: []` to the `out_of_scope` intent so that this isn't treated separately in the future